adapter.go

```diff
@@ -91,11 +104,40 @@ func (a *adapter) open() {
 	}
 
 	db := session.DB(dI.Database)
+
 	collection := db.C("casbin_rule")
 
 	a.session = session
 	a.collection = collection
+	indexes := []string{"ptype", "v0", "v1", "v2", "v3", "v4", "v5"}
+	for _, k := range indexes {
+		if err := a.collection.EnsureIndexKey(k); err != nil {
+			panic(err)
+		}
+	}
+}
 
+func (a *adapter) openByName(name string) {
+	dI, err := mgo.ParseURL(a.url)
+	if err != nil {
+		panic(err)
+	}
+	dI.FailFast = true
+
+	if dI.Database == "" {
+		dI.Database = "casbin"
+	}
+
+	session, err := mgo.DialWithInfo(dI)
+	if err != nil {
+		panic(err)
+	}
+
+	db := session.DB(dI.Database)
+	collection := db.C(name)
```

Comment:
The ``name`` here should be ``"casbin_rule"``, because it's the table name, not the db name.

Reply:
ok, the name really is the table name. I found write the wrong notes when i push,but i have modified,so have three commit.

Reply:
So you should squash your commits into one in this PR. Because there's no need to see how you fix the errors in the commit history.

Reply:
Thank you，I am a rookie. About this name modify, is it still necessary to submit？

Reply:
Please join the Tencent QQ group for discussion: 546057381